### ISSUE TITLE
[bug] REST API returning null responses

### ISSUE THREAD

**Issue by risacher** [wekan/wekan#2342]
## Issue

When I request boards through the REST API in v2.60, some boards just return nothing.  I reverted to v2.55, and the problem went away.  I am using the docker containers 
wekanteam/wekan:v2.55 and wekanteam/wekan:v2.60, 

Code requesting the page is this: 
```
fetch("/wekan/api/boards/"+bId+"/export", {                                                                                             
    method: 'GET',                                                                                                                        
    headers: headers                                                                                                                      
  })                                                                                                                                      
    .then((response) => { return response.json(); })                                                                                      
    .then((rBoard) => {                                                                                                                   
      document.rBoard = rBoard;                                                                                                           
      genPDF(rBoard);                                                                                                                     
    })                                                                                                                                    
  return false;                                                                                                                           
```

Which boards don't return data is consistent, but I haven't yet been able to analyze what is different about the boards that work vs don't.

* Http frontend if any (Caddy, Nginx, Apache, see config examples from Wekan GitHub wiki first):
http frontend is a custom reverse proxy built on node: https://github.com/risacher/yxorp-edon  but I don't think it's relevant.

* Node Version: 8.15.1
* MongoDB Version: mongo: 4.0.5
* ROOT_URL environment variable https://risacher.org/wekan/:

I will try to update with more details to reproduce.

**Comment by risacher** [comment]
narrowing it down slightly: the docker container wekanteam/wekan:v2.56 works okay, but wekanteam/wekan:v2.58 returns zero-length responses (there is no docker tag for 2.57)

**Comment by bentiss** [comment]
I think this should be fixed by #2406 
`export` was broken in v2.58

**Comment by risacher** [comment]
I confirm fixed in v2.73 from hub.docker.  Closing, and thank you.
